wandb/sdk/artifacts/artifact.py

```diff
@@ -510,11 +498,13 @@ def metadata(self, metadata: dict) -> None:
 
     @property
     def ttl(self) -> Union[timedelta, None]:
-        """Time To Live (TTL).
+        """The time-to-live (TTL) policy of an artifact.
 
-        The artifact will be deleted shortly after TTL since its creation.
-        None means the artifact will never expire.
-        If TTL is not set on an artifact, it will inherit the default for its collection.
+        Artifacts are deleted shortly after the TTL policy is passed.
+        `None` means the artifact will never expire.
+        An artifact inherits a TTL policy from the collection it belongs to if
```

Comment:
```suggestion
        If a team admin has [defined a default TTL](https://docs.wandb.ai/guides/artifacts/ttl#set-default-ttl-policies-for-a-team) in the team settings and there is no custom policy set on an artifact, an artifact inherits a TTL policy from the team default. 
```
We don't actually collection level TTL - the only 2 levels where you can set a TTL is at the version level, or at the team level (if your admin has set a team default). So inheritance would only happen from a user's perspective from a team default set by an admin in the team settings.

Reply:
I asked @estellazx about this [here](https://weightsandbiases.slack.com/archives/CT4901KC5/p1706311403293859) btw